core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -435,7 +435,8 @@ class TransactionStateManager(brokerId: Int,
       info(s"Completed loading transaction metadata from $topicPartition for coordinator epoch $coordinatorEpoch")
     }
 
-    scheduler.schedule(s"load-txns-for-partition-$topicPartition", loadTransactions)
+    val nowInMs = time.milliseconds()
```

Comment:
nit: maybe we could call this something more descriptive like `scheduleStartMs`

Reply:
Done